protokube/cmd/protokube/main.go

```diff
@@ -114,6 +114,28 @@ func run() error {
 		if internalIP == nil {
 			internalIP = awsVolumes.InternalIP()
 		}
+	} else if cloud == "digitalocean" {
+		if clusterID == "" {
+			glog.Error("digitalocean requires --cluster-id")
+			os.Exit(1)
+		}
+
+		DOVolumes, err := protokube.NewDOVolumes(clusterID)
```

Comment:
Naming nit: `doVolumes` as I don't think locals are supposed to start with a capital